packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -310,15 +310,22 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         </>
       );
       break;
-    case unsupportedEnvironmentPageTrampoline():
-      pageElement = (
-        <>
-          {
-            /* @conditional-compile-remove(unsupported-browser) */
-            <UnsupportedBrowserPage onTroubleshootingClick={props.options?.onEnvironmentInfoTroubleshootingClick} />
-          }
-        </>
-      );
+  }
+
+  /* @conditional-compile-remove(unsupported-browser) */
+  if (adapter.getState().features?.unsupportedEnvironment) {
```

Comment:
this conditional check shouldn't be necessary, the adapter shouldn't provide a page `unsupportedEnvironment` if the feature isn't opted into